rskj-core/src/main/resources/config/testnet.conf

```diff
@@ -46,6 +46,10 @@ peer {
     networkId = 8100
 }
 
+# the number of milliseconds that should pass from a previous submission of a mining solution before next one is allowed
+# there's no limit, if the number is zero or negative
```

Comment:
I would change it to: "Set this value to zero or negative number to disable this limit"

Reply:
done